Resources/Locale/en-US/_NF/cargo/trade-crate.ftl

```diff
@@ -0,0 +1,3 @@
+trade-crate-destination-station = This crate's destination is [color=springgreen]{$destination}[/color].
+trade-crate-priority-active = This crate is an [color=yellow]express delivery[/color]. It's still on time!
+trade-crate-priority-inactive = This crate is an [color=#886600]express delivery[/color]. It's late.
```

Comment:
Do you think it might make sense to highlight "on time" in the first of these, and potentially "late" in the second? Or make "on time" bold, so it's _extra clear_ that you haven't missed the deadline. :)

Reply:
I'll bold them, but compared to the mail, this has an (admittedly smaller) animated vs. still overlay, and the mail itself has a description that's more subtle.